README.md

```diff
@@ -26,7 +26,56 @@ document**.<br/>
 - [Other Awesome Lists](#other-awesome-lists)
 
 ## Word Processors
-Word Processors which can generate LaTeX, HTML or PDF output on demand.
+
+- [Atom](https://atom.io) - Popular IDE with Markdown support.
+  - [Markdown Preview Enhanced for Atom](https://github.com/shd101wyy/markdown-preview-enhanced) - Pandoc
+   integration and utilities for Atom.
+  - [Autocomplete BibTex for Atom](https://github.com/apcshields/autocomplete-bibtex) - BibLaTeX support for Atom.
+- [Marktext](https://marktext.app/) - Markdown text editor.
+- [R Studio](https://github.com/rstudio/rstudio) - IDE for R.
+  - [bookdown](https://github.com/rstudio/bookdown) - R package to facilitate writing books and long-form articles, reports with R Markdown :bookmark: :link:.
+  - [R Markdown](https://rmarkdown.rstudio.com/) - R package to write R next to Markdown
+   :bookmark:
+   :link:.
+- [Vim](https://www.vim.org/) - Command line text editor.
+- [fzf-bibtex](https://github.com/msprev/fzf-bibtex/#readme) - A BibTeX source
+ with vim integration which uses fzf (a fuzzy finder implemented in Go).
+  -[vim-pandoc](https://github.com/vim-pandoc/vim-pandoc) - Pandoc integration and utilities for Vim.
+  -[vim-pandoc-syntax](https://github.com/vim-pandoc/vim-pandoc-syntax) - Pandoc syntax highlighting for Vim.
```

Comment:
1. fzf-bibtex is not tied to Vim but let's keep it this way unless somebody complains :smile: 
2. Fix indentation:
```suggestion
  - [fzf-bibtex](https://github.com/msprev/fzf-bibtex/#readme) - A BibTeX source
 with vim integration which uses fzf (a fuzzy finder implemented in Go).
  - [vim-pandoc](https://github.com/vim-pandoc/vim-pandoc) - Pandoc integration and utilities for Vim.
  - [vim-pandoc-syntax](https://github.com/vim-pandoc/vim-pandoc-syntax) - Pandoc syntax highlighting for Vim.
```